src/structures/MessageEmbed.js

```diff
@@ -108,6 +110,40 @@ class MessageEmbed {
     while (col.length < 6) col = `0${col}`;
     return `#${col}`;
   }
+
+  /**
+   * The RichEmbed representation of this MessageEmbed.
+   * @type {RichEmbed}
+   * @readonly
+   */
+  get richEmbed() {
+    const result = {};
+    const entries = object => Object.keys(object).map(key => [key, object[key]]);
+    entries(this).map(([key, value]) => {
```

Comment:
You aren't using the value returned from this map. You should be using forEach() for speed instead.